config.toml.example

```diff
@@ -502,12 +502,15 @@
 # only use static libraries. If unset, the target's default linkage is used.
 #crt-static = false
 
-# The root location of the MUSL installation directory. The library directory
+# The root location of the musl installation directory. The library directory
 # will also need to contain libunwind.a for an unwinding implementation. Note
-# that this option only makes sense for MUSL targets that produce statically
+# that this option only makes sense for musl targets that produce statically
 # linked binaries
 #musl-root = "..."
 
+# The location of musl libdir.
```

Comment:
Could you add a note here that the location should include the musl-root as well? Alternatively, I'm not sure if we should actually require that -- maybe we should prepend the root instead? Is it typical to have the libdir outside the root?

Reply:
Mentioned that this is a full path. The idea was that this should be the same as `./configure --libdir=...` used with musl.